intelmq/lib/bot.py

```diff
@@ -245,7 +265,11 @@ def acknowledge_message(self):
         self.last_message = self.current_message
         self.source_pipeline.acknowledge()
 
-    def dump_message(self):
+    def dump_message(self, error_traceback):
+        if self.current_message is None:
```

Comment:
hum... in my perspective, dump file is for messages that have some 'bug' and error traceback is just the context to help sysadmin to fix the message. So, if no current_message defined, no dump log is need.

Do you agree @sebix ? 

May be we can try to find a way to define current_message in collector bots case... do you see any reasoable approach?